_posts/2025-04-22-OpenSearch-version-3-features.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "OpenSearch® version 3.0: New features, breaking changes from Apache Lucene™ 10, and what to expect"
+authors:
+ - kwren
+date: 2025-04-22
+categories:
+ - technical-post
+meta_keywords: 
+meta_description:
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ OpenSearch 3.0, Apache Lucene 10, JVM 21, breaking changes, performance improvements, Java Security Manager, JavaScript client, SQL plugin, sparse indexing, KNN search, neural search, OpenSearch dashboard

_meta_description:_ Dive into the major updates in OpenSearch 3.0, including Lucene 10 integration, JVM 21 adoption, and performance improvements. Learn about breaking changes, critical upgrades, and enhanced capabilities.